lib/Runtime/PlatformAgnostic/ICU.h

```diff
@@ -8,7 +8,15 @@
 
 #ifdef HAS_ICU
 #ifdef WINDOWS10_ICU
+// if WINDOWS10_ICU is defined, pretend like we are building for recent Redstone,
+// even if that isn't necessarily true
+#ifdef NTDDI_VERSION
+#undef NTDDI_VERSION
+#endif
+#define NTDDI_VERSION NTDDI_WIN10_RS3
```

Comment:
Does this mean we do require RS3?

Reply:
I think we actually require RS2 from the offline conversation I've had with Jack, but pretending RS3 in this case is effectively the same?

Reply:
the Windows Kit ICU headers are gated on NTDDI_VERSION >= RS2. I suppose RS3 was somewhat arbitrary, but the idea is that we need to trick the Kit headers into thinking that the corresponding ICU DLLs will be available on the system (we know they will because of the earlier Chakra.ICU work)